website/content/docs/cloudinary.md

```diff
@@ -30,55 +30,101 @@ media_library:
 
 ## Image transformations
 
-The Cloudinary integration allows images to be transformed in two ways: directly within Netlify CMS, and separately from the CMS via Cloudinary's [dynamic URL's](https://cloudinary.com/documentation/image_transformations#delivering_media_assets_using_dynamic_urls). If you transform images within the Cloudinary media library, the transformed image URL will be output by default. This gives the editor complete freedom to make changes to the image output.
+The Cloudinary integration allows images to be transformed in two ways: directly within Netlify CMS via [Cloudinary's Media Library](#transforming-images-via-media-library), and separately from the CMS via Cloudinary's [dynamic URL's](https://cloudinary.com/documentation/image_transformations#delivering_media_assets_using_dynamic_urls) by [inserting cloudinary urls](#inserting-cloudinary-url-in-page-templates).
 
-## Art direction and responsive images
+### Transforming images via Media Library
+If you transform and insert images from within the Cloudinary media library, the transformed image URL will be output by default. This gives the editor complete freedom to make changes to the image output.
+There are two ways to configure image transformation via media library - globally and per field. Global options will be overridden by field options.
 
-If you prefer to provide art direction so that images are transformed in a specific way, or dynamically retrieve images based on viewport size, you can do so by providing your own base Cloudinary URL and only storing the asset filenames in your content:
+#### Global configuration
 
-1. Either globally or for specific fields, configure the Cloudinary extension to only output the asset filename:
+Global configuration, which is meant to affect the Cloudinary widget at all times, can be provided
+as seen below, under the primary `media_library` property. Settings applied here will affect every
+instance of the Cloudinary widget.
 
 ```yml
 # global
 media_library:
   name: cloudinary
-  output_filename_only: true
+  output_filename_only: false
+  default_transformations:
+    -   - fetch_format: auto
+          width: 160
+          quality: auto
+          crop: scale
+```
+
+#### Field configuration
+
+Configuration can also be provided for individual fields that use the media library. The structure
+is very similar to the global configuration, except the settings are added to an individual `field`.
+For example:
 
+```yml
 # field
-fields:
-  - { name: image, widget: image, media_library: { output_filename_only: true } }
+fields: # The fields each document in this collection have
+      - label: 'Cover Image'
+        name: 'image'
+        widget: 'image'
+        required: false
+        tagname: ''
+        media_library:
+          config:
+            default_transformations:
+              -   - fetch_format: auto
```

Comment:
Is the nested array intentional here?

Reply:
Yes, this is intentional. The default_transformation type should be `Object[][]`.